### ISSUE TITLE
False triggering of RCS1089

### ISSUE THREAD

**Issue by jnyrup** [JosefPihrt/Roslynator#123]
```csharp
class MyClass
{
    public int Property { get; set; }
}

int Property = 1;
var cls = new MyClass()
{
	Property = Property + 1 // <-- 
};
```

RCS1089 is triggered suggestion to replace it with `Property++`.
But the left hand side of the addition is not the same node as the left hand side of the assignment.

This seems related to #19
